pkg/command/plugin.go

```diff
@@ -431,12 +431,24 @@ func displayInstalledPluginList(installedPlugins []cli.PluginInfo, writer io.Wri
 		//    context = installedPlugins[index].ContextName
 		// }
 
+		status := common.PluginStatusNotInstalled
+		if installedPlugins[index].Status == common.PluginStatusInstalled ||
+			installedPlugins[index].Status == common.PluginStatusUpdateAvailable {
+			// TODO(khouzam): For the moment, we don't show a status of "update available"
```

Comment:
question: I agree with reasoning for status shown as `Installed` instead of `updates available`. Is it something we are planning to fix for 1.0.0 release?

Reply:
It is something we would like to fix for 1.0.  However, we haven't discussed a solution yet.
For example, if I'm running a TKG 1.6, and the Central Repo contains all plugins, including for TKG 2.1, do I really want to be told an upgrade is available?  That upgrade is not applicable to my TKG version.

So the problem is a little tricky and will need some thought.

Reply:
Okay with the change now to reduce the confusion. 
Can you file a github issue to track that we need to revise the update notification UX.
And after we do that, perhaps both comments can be simplified to just point to the issue,
and we can elaborate more in the issue description instead.

Reply:
I have opened #65 and reduced both comments of the code to point to it.